ios/Scripts/quick_icon_generator.py

```
@@ -0,0 +1,91 @@
+    return success == len(sizes)
+
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
+        print("Использование: python quick_icon_generator.py path/to/your/icon.png")
+        sys.exit(1)
+
+    source_path = sys.argv[1]
+
+    try:
+        from PIL import Image
```

Flake8 (7.2.0)

[error] redefinition of unused 'Image' from line 9

(F811)